vast/vast.cpp

```diff
@@ -140,7 +140,14 @@ int main(int argc, char** argv) {
       }
     }
     if (!plugin_found) {
-      std::cerr << "failed to find plugin: " << plugin_file << std::endl;
+      std::cerr << "failed to load plugin: " << plugin_file;
```

Comment:
Here is what I think the outer loop should consist of:
```
  for (const auto& plugin_file : plugin_files) {
    if (auto err = load_plugin(plugin_file))
      std::cerr << "failed to load plugin " << plugin_file << ':' << render(err)
                << std::endl;
  }
```
The iterating over candidate directories should not leak out of `load_plugin`.

Reply:
I'm not against that and will implement this now because I need to get this merged, but please for the next time write a follow-up story because that logic change is entirely unrelated to this PR.

Reply:
I pushed a commit. Please review again on the sooner side.